src/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "trading-bot-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.mjs",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./app.mjs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/NJain0001/trading-bot-api.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/NJain0001/trading-bot-api/issues"
+  },
+  "homepage": "https://github.com/NJain0001/trading-bot-api#readme",
+  "dependencies": {
```

Comment:
Doesn't dotenv also need to be here?

Reply:
added the missing package to the json file